tests/functional/upgrade/test_non_stretch_cluster_upgrade.py

```
@@ -0,0 +1,529 @@
+    "pod_obj_list": None,
+    "outage_node_name": None,
+    "md5sum_before": None,
+}
+
+
+# ---------------------------------------------------------------------------
+# Module-level helpers
+# ---------------------------------------------------------------------------
+
+
+def _pods_for_pvcs(pod_obj_list, wait=True):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_pods_for_pvcs`

(ANN202)